packages-native/popup-menu/README.md

```diff
@@ -1,76 +1,92 @@
-## PopupMenu
+## Pop-Up Menu
 
-Use pop up menu to create a menus which anchored to an element. Pop up menu will be opened after pressing to the anchor element and close when one of the items are clicked.
+​
+Use Pop-Up Menu to create a menu which is anchored to an element. The pop-up menu will be opened after clicking on the anchor element, and will close when one of the items are clicked.
```

Comment:
Would be better to or use all capitalized or not.. After first phrase pop-up menu is lowercase

Reply:
This went through Connor's hands but i can discuss with him ofc